cog/daily_charge.py

```
@@ -23,95 +23,257 @@
+
+    def embed_successful(self, point, combo, user: discord.User | discord.Member):
         # 讀表符ID
         with open(
             f"{os.getcwd()}/DataBase/server.config.json", "r", encoding="utf-8"
         ) as file:
             stickers = json.load(file)["SCAICT-alpha"]["stickers"]
 
-        self.embed = discord.Embed(
-            title=f"{interaction.user.name}剛剛充電了！", description="", color=0x14E15C
+        embed = discord.Embed(
+            title=f"{user.name}剛剛充電了！",
```

Ruff (0.8.2)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)